server/enterprise/license.go

```diff
@@ -0,0 +1,20 @@
+package enterprise
+
+import (
+	pluginapi "github.com/mattermost/mattermost-plugin-api"
+	"github.com/mattermost/mattermost-server/v5/model"
+)
+
+const (
+	e20          = "E20"
```

Comment:
nit: can these not be imported from a server package?

Reply:
@levb They exist as private constants in the `mattermost-plugin-api`. I don't see them in `mattermost-server`.

I think the right thing to do here (and in Jira plugin https://github.com/mattermost/mattermost-plugin-jira/pull/775) is to actually update the `mattermost-plugin-api` package to include this check here. It's just missing a permutation of helper functions. Should be something like `IsEnterpriseAboveE10OrConfiguredForDevelopment()`, or `isE10ProfessionalEnterpriseOrConfiguredForDevelopment()`. That is what we would be using for any other project that uses the source available license that excludes E10 anyway.

Reply:
https://github.com/mattermost/mattermost-plugin-api/blob/02961f034e242600cdc3ef99a554c06c8b70b3dc/license.go#L67

Reply:
@levb Do you think it's a big deal to have these hardcoded here, or should we update the helpers in [mattermost-plugin-api](https://github.com/mattermost/mattermost-plugin-api/blob/02961f034e242600cdc3ef99a554c06c8b70b3dc/license.go#L67) to include the functionality we need (to exclude only E10)?